test/runtests.jl

```diff
@@ -26,7 +26,7 @@ max_worker_rss != typemax(Csize_t) && move_to_node1("parallel")
 cd(dirname(@__FILE__)) do
     n = 1
     if net_on
-        n = min(8, Sys.CPU_CORES, length(tests))
```

Comment:
> I wonder if there are systems with lots of cores and too little memory

FWIW, HiKey board can have 8 cores and 1G of memory (the situation is common for ARM boards), which is likely not enough to run the test (even with a low `JULIA_TEST_MAX_RSS`) on all cores but an arbitrary worker number limit won't help with this either......